src/api.js

```diff
@@ -33,8 +33,9 @@ class API {
     this.user_id = null;
     this.access_token = null;
     this.entitlements_token = null;
-    this.user_agent = "RiotClient/43.0.1.4195386.4190634 rso-auth (Windows; 10;;Professional, x64)";
-    this.client_version = "release-05.00-shipping-6-725355";
+    this.user_agent =
+      "RiotClient/58.0.0.4640299.4552318 %s (Windows;10;;Professional, x64)";
```

Comment:
what's up with the unused format string `%s`? assuming this is meant to be `rso-auth`, or is the latest version of the official app sending `%s` by mistake as well?

Reply:
also can you put on the same line please :)

Reply:
I checked again there is actually a %s in the user_agent of the riot client and yes we can put it in one line the only reason why it is not is by the code formatter I have installed in VS Code

Reply:
Nevermind I fucked up there is actually no unused format string it has to be rso-auth thanks for pointing this out

Reply:
I committed a new version of the user-agent